src/bundle/Resources/views/themes/admin/content_type/field_types.html.twig

```diff
@@ -129,7 +129,7 @@
                 data-universaldiscovery-title="{{ "field_definition.ezobjectrelation.selection_root_udw_title"
                 |trans({}, "content_type")
                 |desc("Select where to start browsing for a Relation") }}"
-                class="ez-button-tree pure-button ez-font-icon ez-button ez-relation-pick-root-button btn btn-secondary btn--udw-relation-default-location"
```

Comment:
Can we delete these? They seem to be unused. ping @dew326 @lucasOsti @GrabowskiM

Reply:
It seems ok to remove it.